activesupport/lib/active_support/core_ext/module/attribute_accessors.rb

```diff
@@ -91,7 +91,7 @@ def #{sym}
   #   Person.new.hair_colors = [:blonde, :red]
   #   HairColors.class_variable_get("@@hair_colors") # => [:blonde, :red]
   #
-  # If you want to opt out the instance writer method, pass
+  # If you want to opt out of the creation of the instance writer method, pass
```

Comment:
=> `creation of ` we are using this differently at many places, with or without

Reply:
I didn't get you. Did you mean to say we don't need the "the" before "creation" or something else?